client/src/components/FilterMenu.js

```diff
@@ -1,16 +1,18 @@
 import React, { Component } from 'react';
-import PropTypes from 'prop-types';
-import PlacesAutocomplete, {
-  geocodeByAddress,
-  getLatLng,
-} from 'react-places-autocomplete';
+import { bool, func } from 'prop-types';
+import PlacesAutocomplete, { geocodeByAddress, getLatLng } from 'react-places-autocomplete';
 import '.././css/FilterMenu.css';
 
+const propTypes = {
+  visibility: bool.isRequired,
+  changeSalary: func.isRequired,
+  changeLocation: func.isRequired,
+};
+
 class FilterMenu extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      isMenuVisible: false,
```

Comment:
Seems like this isn't used and we're already passing in a visibility prop anyway.